src/lib/card/card.ts

```diff
@@ -40,40 +36,32 @@ declare global {
  *
  * @csspart root - The root container element.
  *
+ * @state raised - The state of the card when raised.
+ *
  * @cssclass forge-card - The card container element _(required)_.
  * @cssclass forge-card--raised - The card container element when raised _(required)_.
  */
-@customElement({
-  name: CARD_CONSTANTS.elementName
-})
-export class CardComponent extends BaseComponent implements ICardComponent {
-  public static get observedAttributes(): string[] {
-    return Object.values(CARD_CONSTANTS.observedAttributes);
-  }
+@customElement(CARD_TAG_NAME)
+export class CardComponent extends LitElement implements ICardComponent {
+  public static styles = unsafeCSS(styles);
+
+  #internals: ElementInternals;
 
-  private _raised = false;
+  /** Whether the card has elevation or not. */
+  @property({ type: Boolean, reflect: true }) public raised = false; // TODO: remove reflect in v4 in favor of :state
```

Comment:
Supporting attribute reflection and state until we can fully phase out reflection seems like the right path 👍